posthog/temporal/batch_exports/redshift_batch_export.py

```diff
@@ -53,6 +53,23 @@
 from posthog.temporal.common.heartbeat import Heartbeater
 from posthog.temporal.common.logger import configure_temporal_worker_logger
 
+NON_RETRYABLE_ERROR_TYPES = [
+    # Raised on errors that are related to database operation.
+    # For example: unexpected disconnect, database or other object not found.
+    "OperationalError",
+    # The schema name provided is invalid (usually because it doesn't exist).
+    "InvalidSchemaName",
+    # Missing permissions to, e.g., insert into table.
+    "InsufficientPrivilege",
+    # A column, usually properties, exceeds the limit for a VARCHAR field,
+    # usually the max of 65535 bytes
+    "StringDataRightTruncation",
+    # Raised by our PostgreSQL client when failing to connect after several attempts.
+    "PostgreSQLConnectionError",
+    # Column missing in Redshift, likely the schema was altered.
+    "UndefinedColumn",
+]
```

Comment:
Err, didn't intend to get this out yet, working on SPMC for redshift. But anyways, won't hurt to merge now.